### PR TITLE
added toggle buttons

### PR DESCRIPTION
**Pull request by Dishebh**
fixes issue: #644 

Added toggle buttons in `Molecules` on `/interface/toggleButtons` route.

### CODE REVIEW

**Comment by Dishebh** [comment]
> Consider use cases and make it dynamic accordingly.
> 
> For example, I would expect these `toggle buttons` to accept `leftLabel or leftIcon` and `rightLabel or rightIcon` as props. In addition click event would also change `category` of buttons accordingly.
> 
> Complete those first. Let me know if you have any doubts or confusion.

Added the props, pls see if there are any more changes.

File: src/ignitus-Shared/ignitus-DesignSystem/ignitus-Molecules/ignitus-ToggleButtons/Components/index.tsx [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+  margin: 0;
+  border: 0;
+  float: left;
+  border-radius: 0 20px 20px 0;
+`;
+
+export const DefaultToggleButton1 = ({
+  leftLabel,
+  rightLabel,
+}: InputTypes1) => (
```
Comment (Marvin9) [inline src/ignitus-Shared/ignitus-DesignSystem/ignitus-Molecules/ignitus-ToggleButtons/Components/index.tsx:27]:
```suggestion
export const DefaultToggleButton1: React.FC<InputTypes1> = ({
  leftLabel,
  rightLabel,
}) => (
```

Comment (Dishebh) [reply src/ignitus-Shared/ignitus-DesignSystem/ignitus-Molecules/ignitus-ToggleButtons/Components/index.tsx:27]: Which category shall I assign the button, when it is selected?

**Comment by Dishebh** [comment]
> Good work.
> 
> Now make buttons `category` dynamic. Track which one is selected and set `category` accordingly.

Which category shall I assign the button, when it is selected?

**Comment by Dishebh** [comment]
> Good work.
> 
> Now make buttons `category` dynamic. Track which one is selected and set `category` accordingly.

@Marvin9 Should I have to use react hooks here, to change the `state` of the button `onClick`?

**Comment by Marvin9** [comment]
Yes you have to use ```useState```. 

Category should be what it is right now as initial & vice versa on click event.

**Comment by Dishebh** [comment]
> Yes you have to use `useState`.
> 
> Category should be what it is right now as initial & vice versa on click event.

Added `state` to the component.

**Comment by Dishebh** [comment]
> Go through this PR and merge it. [Dishebh#3](https://github.com/Dishebh/Ignitus-client/pull/3)

Thank you so much for this! :) Look amazing :rocket:

**Comment by Dishebh** [comment]
@divyanshu-rawat can you please review?

**Pull request merged**